recipe/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+
+class CategoryRecipe(models.Model):
+    name = models.CharField(max_length=10)
+
+    class Meta:
+        db_table = 'categories_recipes'
```

Comment:
여기는 이름만 수정해주시면 좋을 것 같아요~

```suggestion
class RecipeCategory(models.Model):
    name = models.CharField(max_length=10)

    class Meta:
        db_table = 'categories_recipes'
```

Reply:
수정완료했습니다.